_jobs/ux-ui-designer-researcher.md

```diff
@@ -59,5 +59,4 @@ You’ll collaborate with a small team of designers and engineers. You’ll have
 
 ## To Apply
 
-Send resume, brief intro email and preferences for working (hours, time zone, etc) to [careers@compiler.la](mailto:careers@compiler.la) by July 1st 2022.
-Cover letters are not expected and extra weight in scoring will not be given to those that include them.
+Please apply using [this form](https://forms.clickup.com/8631512/f/87d6r-14154/EYTB48KL0KYX71LH1Q?Job%20Role=UX%2FUI%20Designer). We look forward to hearing from you!
```

Comment:
Ah sorry @mrose914 I should have been more clear:

The link should be added to the top "front matter" (everything between `---`) like:

```
---
type: Contract
apply_link: https://forms.clickup.com/8631512/f/87d6r-14154/EYTB48KL0KYX71LH1Q?Job%20Role=UX%2FUI%20Designer
---
```

This makes the "Apply" button show up on the post page and link to the form (and only during the job open window).

Check out the SE post for an example: https://github.com/compilerla/compiler.la/blob/main/_jobs/software-engineer.md